src/wp-admin/security.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Security Administration Screen.
+ *
+ * @package ClassicPress
+ * @subpackage Administration
+ */
+
+/** ClassicPress Administration Bootstrap */
+require_once( dirname( __FILE__ ) . '/admin.php' );
+
+$title = __('Security');
+
+require_once( ABSPATH . 'wp-admin/admin-header.php' );
+
+$tabs = [
+    'users' => [
+        'title' => __('For Users'),
+        'classes' => ['nav-tab']
+    ],
+    'developers' => [
+        'title' => __('For Developers'),
+        'classes' => ['nav-tab']
+    ]
+];
+$active_tab = (in_array(@$_GET['tab'], array_keys($tabs)))
+    ? $_GET['tab']
+    : 'users';
+$tabs[$active_tab]['classes'][] = 'nav-tab-active';
+
+?>
+<div class="wrap">
+  <h1><?php echo esc_html( $title ); ?></h1>
+  <h2 class="nav-tab-wrapper">
+<?php   foreach ($tabs as $tab => $info): ?>
+    <a href="/wp-admin/security.php?tab=<?=$tab?>" class="<?=join(' ', $info['classes'])?>"><?=$info['title']?></a>
```

Comment:
PHP short tags used a couple of times here plus absolute url `/wp-admin/security.php` for a url which will probably break if CP is installed in a folder other than the root domain.

Reply:
@invisnet has fixed this, thanks!